### ISSUE TITLE
Multi-GPU Training Hang on Forever

### ISSUE THREAD

**Issue by lsyysl9711** [lucidrains/vector-quantize-pytorch#141]
Thanks for sharing the codes! It is a great repo.

However, when I try to move my codes into multi-gpu setting, the code will be stucked forever in the accelerator.backward(loss). My framework is very simple: just a normal transformer optimized by reinforcement learning.

My version of VQP is: 1.0.7. GPU is two 3090 with CUDA 11.6 (torch 1.13). 

After I check some similar issues on pytorch's repo, I find that this maybe due to the driver of Nvidia. But I don't know how to fix it, maybe I should change my current machines with a new version of driver?

**Comment by lsyysl9711** [comment]
I find the solution that works for me!

https://github.com/lucidrains/magvit2-pytorch/issues/18#issuecomment-2097405572

I hope this could do some help to people who meet similar problems!
